### PR TITLE
Replace tab with sphinx design

### LINKED ISSUE #182: Migrate to sphinx-design
🐞 **Problem**
The [sphinx-design] library offers lots of features and new capabilities that we can take advantage of. For example, it includes tabs and panels among other features.

This means that we can get rid of [sphinx-panels] and [sphinx-tabs] by having all this features collected under the same library.

In addition, this may allow us to solve the dependencies issue in #135 and finally migrate to latest Sphinx version.

[sphinx-design]: https://sphinx-design.readthedocs.io/en/latest/
[sphinx-panels]: https://sphinx-panels.readthedocs.io/en/latest/
[sphinx-tabs]: https://sphinx-tabs.readthedocs.io/en/latest/

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Fix #182 . Adding here a css file for sphinx design, and can add these feature in ``ansys-sphinx-theme`` later.

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]
Excelent work, @Revathyvenugopal162. Thanks a lot for implementing this.

My only suggestion is about the size of the titles in the tabs. Those look a bit big if compared to the surrounding text but it is not an issue.

**Comment by jorgepiloto** [comment]
Feel free to merge this when you consider, @Revathyvenugopal162 🚀

File: doc/source/_static/css/ansys.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    outline: none;
+    font-size: large;
+    -webkit-tap-highlight-color: transparent;
+    color: var(--pst-color-text-base);
+}
+
+
+.sd-tab-set>input:checked+label+.sd-tab-content {
+    display: block;
+    font-size: large;
```
Comment (Revathyvenugopal162) [inline doc/source/_static/css/ansys.css:14]:
```suggestion
    font-size: medium;
```

**Comment by MaxJPRey** [comment]
@Revathyvenugopal162 This looks really good. Thanks for implementing it.

**Pull request merged**